docs/resources/ruleset.md

```diff
@@ -483,6 +483,7 @@ Read-Only:
 Optional:
 
 - `algorithms` (Block List) Compression algorithms to use in order of preference. (see [below for nested schema](#nestedblock--rules--action_parameters--algorithms))
+- `additional_cacheable_ports` (Set of Int) Specifies additional cacheable ports for use by Origin servers
```

Comment:
the documentation is generated automatically based on the schema description (see https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md)